docs/fidesops/docs/guides/configuration_reference.md

```diff
@@ -86,6 +89,9 @@ TASK_RETRY_DELAY=20
 TASK_RETRY_BACKOFF=2
 REQUIRE_MANUAL_REQUEST_APPROVAL=True
 MASKING_STRICT=True
+
+[user]
+ANALYTICS_OPT_OUT=True
```

Comment:
You probably don't want to encourage opt-outs by including `True` as the value in your example config...